src/lib/wallets/lobstr/__test__/lobstr.test.ts

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { expect } from '@jest/globals';
+import { signTransaction } from '@lobstrco/signer-extension-api';
+import * as StellarSdk from 'stellar-sdk';
+
+import { StellarNetwork } from '../../../stellar/StellarNetwork';
+import LocalStorage from '../../../storage/storage';
+import Lobstr from '../Lobstr';
+
+const signedXdr =
+    'AAAAAgAAAAA2jYMwhev3yM7P+JWOv6kRQZAssek5zytAbbyhJbOjNQAAAGQAATOSAAAAAQAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAA2jYMwhev3yM7P+JWOv6kRQZAssek5zytAbbyhJbOjNQAAAAAF9eEAAAAAAAAAAAA=';
+const mockTx = {
```

Comment:
I think this is not well-formatted
```suggestion
const signedXdr = 'AAAAAgAAAAA2jYMwhev3yM7P+JWOv6kRQZAssek5zytAbbyhJbOjNQAAAGQAATOSAAAAAQAAAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAAA2jYMwhev3yM7P+JWOv6kRQZAssek5zytAbbyhJbOjNQAAAAAF9eEAAAAAAAAAAAA=';

const mockTx = {
```